src/store/actions/streamsAction.js

```diff
@@ -1,4 +1,11 @@
-import { FETCH_STREAMS_REQUEST, FETCH_STREAMS_SUCCESS, FETCH_STREAMS_FAILURE } from '../actionTypes';
+import {
+  FETCH_STREAMS_REQUEST,
+  FETCH_STREAMS_SUCCESS,
+  FETCH_STREAMS_FAILURE,
+  ADD_NEW_STREAM_FAILURE,
+  ADD_NEW_STREAM_SUCCESS,
+  FETCH_NEW_STREAM_REQUEST,
+} from '../actionTypes';
 import axios from '../../axiosApi';
 
```

Comment:
Приложение падает из за того что не импортирвал NotificationManager
```suggestion
import { NotificationManager } from 'react-notifications';
```
![зау](https://user-images.githubusercontent.com/86306282/132995036-a2225894-e393-4855-8963-e44ddafd1b4d.png)

Reply:
@BauyrjanSeidahmet Поправил